# Nemabiome

This repository contains the code for the Nemabiome website and is built entirely with Rmarkdown.

## What is the nemabiome?

Deep amplicon sequencing, or metabarcoding, using next-generation sequencing platforms, has revolutionized the study of microbial communities in humans, animals and the environment. Like microbes, parasites often exist in complex communities within a host and/or the external environment. We have coined the phrase “nemabiome” to describe the community of nematodes that inhabit a single host animal or environmental niche. We have recently described the use of deep amplicon sequencing, targeting the internal transcribed spacer 2 (ITS-2) rDNA locus, to study the gastro-intestinal nemabiome of cattle (Avramenko et al 2015, Avramenko et al 2017). Nemabiome sequencing provides a detailed picture of the species composition of the GIN parasite community structure in large sample sets and has a huge number of potential applications in diagnostics, surveillance and research. This website provides an overview of the methods we currently use for nemabiome sequencing, targeting the ITS-2 rRNA locus, in a number of host species including sheep, bison and horses. We plan to update this website as our methods, applications and targeted loci evolve. The aim is to enable any laboratory with basic parasitology and molecular biology capacity to undertake the approach.

## Contributors

* The nemabiome concept and development of tools to study the nemabiome is done in the lab of [John Gilleard](http://www.ucalgary.ca/jsgilleard/) at the University of Calgary in the Faculty of Veterinary Medicine
* Bench protocols written by Elizabeth Redman
* Analysis protocols written by Russell Avramenko
* ITS2 database and taxonomy constructed by Russell Avramenko
* Nemabiome logo design by Matilde Tonelli
* Website built by Matthew Workentine, Bioinformatics Consultant, Faculty of Veterinary Medicine