<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">

<head>

<meta charset="utf-8">
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="pandoc" />




<title></title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/paper.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-1.1/default.css" rel="stylesheet" />
<script src="site_libs/highlightjs-1.1/highlight.js"></script>
<link href="site_libs/ionicons-2.0.1/css/ionicons.min.css" rel="stylesheet" />
<link href="https://fonts.googleapis.com/css?family=Nunito+Sans|Roboto|Lobster" rel="stylesheet">

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs && document.readyState && document.readyState === "complete") {
   window.setTimeout(function() {
      hljs.initHighlighting();
   }, 0);
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>

<link rel="stylesheet" href="styles.css" type="text/css" />

</head>

<body>

<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
  height: auto;
}
.tabbed-pane {
  padding-top: 12px;
}
button.code-folding-btn:focus {
  outline: none;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 64px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 69px;
  margin-top: -69px;
}

.section h2 {
  padding-top: 69px;
  margin-top: -69px;
}
.section h3 {
  padding-top: 69px;
  margin-top: -69px;
}
.section h4 {
  padding-top: 69px;
  margin-top: -69px;
}
.section h5 {
  padding-top: 69px;
  margin-top: -69px;
}
.section h6 {
  padding-top: 69px;
  margin-top: -69px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>


<div class="container-fluid main-container">

<!-- tabsets -->
<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});
</script>

<!-- code folding -->





<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">
      	<img alt="Nemabiome Logo" src="images/logo/nemabiome_compact_64.png" style="width:35px;height:35px">
      </a>
    </div>
    
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
				
				<li>
					<a href="overview.html">
						<span class="ion ion-map"></span>
    				Overview
					</a>
				</li>
	
				<li class="dropdown">
  				<a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">
    				<span class="ion ion-erlenmeyer-flask"></span>
    				Protocols
     				<span class="caret"></span>
     			</a>
  				<ul class="dropdown-menu" role="menu">
      			<li><a href="parasite.html">Parasite</a></li>
   					<li><a href="bench.html">Sequencing</a></li>
            <li><a href="analysis.html">Analysis</a></li>
  				</ul>
				</li>
			
				<li>
					<a href="examples.html">
						<span class="ion ion-pie-graph"></span>
						Worked Example
					</a>
				<li>

				</li>
				<li>
  				<a href="citation.html">
						<span class="ion ion-document-text"></span>
    		    Citation
  				</a>
				</li>
				<li>
  				<a href="about.html">
    				<span class="ion ion-information-circled"></span>
    		    About
  				</a>
				</li>
    	</ul>
      
      <ul class="nav navbar-nav navbar-right">

      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div class="fluid-row" id="header">




</div>


<div id="sequencing-protocols" class="section level1 tabset tabset-pills tabset-fade">
<h1>Sequencing Protocols</h1>
<p>Download a pdf of all the protocols for printing <a href="downloads/sequencing_protocols.pdf">here</a></p>
<div id="template-preparation" class="section level2">
<h2>Template preparation</h2>
<div id="reagents-required" class="section level3">
<h3>Reagents required</h3>
<ul>
<li>lysis buffer (Made in-house, see recipe below)</li>
<li>proteinase K (20 mg/ml)</li>
<li>thermomixer</li>
<li>1.5 ml eppendorf tubes</li>
</ul>
<?xml version="1.0" encoding="UTF-8"?>
<table class="table table-striped table-hover table-condensed table-responsive" style="width: auto !important; text-align: right;">
<caption>
Lysis buffer (Total vol 50ml)
</caption>
<thead>
<tr>
<th style="text-align:left;">
volume (<span class="math inline">\(\mu\)</span>l)
</th>
<th style="text-align:left;">
stock Reagent
</th>
<th style="text-align:left;">
concentration in lysis buffer
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
2500
</td>
<td style="text-align:left;">
1M KCl
</td>
<td style="text-align:left;">
50mM KCl
</td>
</tr>
<tr>
<td style="text-align:left;">
500
</td>
<td style="text-align:left;">
1M Tris (pH 8.3)
</td>
<td style="text-align:left;">
10mM Tris (pH 8.3)
</td>
</tr>
<tr>
<td style="text-align:left;">
125
</td>
<td style="text-align:left;">
1M MgCl<span class="math inline">\(_2\)</span>
</td>
<td style="text-align:left;">
2.5mM MgCl<span class="math inline">\(_2\)</span>
</td>
</tr>
<tr>
<td style="text-align:left;">
225
</td>
<td style="text-align:left;">
Nonidet p-40
</td>
<td style="text-align:left;">
0.45% Nonidet p-40
</td>
</tr>
<tr>
<td style="text-align:left;">
225
</td>
<td style="text-align:left;">
Tween-20
</td>
<td style="text-align:left;">
0.45% Tween-20
</td>
</tr>
<tr>
<td style="text-align:left;">
250
</td>
<td style="text-align:left;">
2% gelatin
</td>
<td style="text-align:left;">
0.01% gelatin
</td>
</tr>
</tbody>
<tfoot>
<tr>
<td style="padding: 0; border:0;" colspan="100%">
<sup>*</sup> Make-up to 50 ml with mol grade water. Autoclave, aliquot and store at -20
</td>
</tr>
</tfoot>
</table>
</div>
<div id="protocol" class="section level3">
<h3>Protocol</h3>
<ol style="list-style-type: decimal">
<li>UV treat plasticware and mol grade water for 15 min</li>
<li>Transfer 1000 ethanol-fixed L3s (or L1s or eggs) to 1.5 ml tube</li>
<li>Add lysis buffer to achieve a vol 1.4 ml and incubate at room temp for 5 min</li>
<li>Centrifuge at 13000 G for 4 min to pellet parasite material</li>
<li>Remove and discard majority of supernatent (do not touch pellet)</li>
<li>Add 1 ml lysis buffer and re-suspend pellet with pipet</li>
<li>Repeat steps 4-6 twice more</li>
<li>Centrifuge at 13000 G for 4min to pellet parasite material</li>
<li>Remove and discard supernatant to leave approx 100 <span class="math inline">\(\mu\)</span>l</li>
<li><p>Re-suspend pellet and add another 50 <span class="math inline">\(\mu\)</span>l of lysis buffer</p></li>
<li><p>If lysing L3s place on thermomixer for 15 min at 95 with shaking (1000 RPM)<br />
<em>This helps with the break down of the L3s tough outer sheath</em></p></li>
<li>Place at <span class="math inline">\(-80\,^{\circ}{\rm C}\)</span> for a min of 1hour to allow ice crystals to help with the degradation process</li>
<li>Allow to de-frost on ice</li>
<li>Add 6 <span class="math inline">\(\mu\)</span>l Proteniase K (20 mg/ml) to each sample to achieve a final conc of 0.8 mg/ml in 150 <span class="math inline">\(\mu\)</span>l</li>
<li>Place on thermomixer at 55 for 120 min with shaking at 800 RPM followed by 20 min at 95 to denature the proteinase K</li>
<li>Place directly on ice and make a 1:10 lysate dilution (with mol grade water) in a new tube for using as template for PCR</li>
<li><p>Store both lysate and lysate dilution at <span class="math inline">\(-80\,^{\circ}{\rm C}\)</span><br />
<em>Storage at <span class="math inline">\(-80\,^{\circ}{\rm C}\)</span> is critical due to the crude nature of the DNA preparation as quality of template will degrade over time at <span class="math inline">\(-10\,^{\circ}{\rm C}\)</span></em></p></li>
</ol>
</div>
</div>
<div id="library-preparation" class="section level2">
<h2>Library preparation</h2>
<div id="amplication-of-its-2-region" class="section level3">
<h3>Amplication of ITS-2 region</h3>
<div id="reagents-required-1" class="section level4">
<h4>Reagents required</h4>
<ul>
<li>Kapa HiFi Hotstart PCR kit with dNTPs (Kapa Biosystems, KK2502)</li>
<li><ol start="0" style="list-style-type: decimal">
<li>2ml 96-well plates, clear, half-skirted (VWR, 83007-374)</li>
</ol></li>
<li>Microseal “A” film (Biorad, MSA-5001)</li>
<li>1.5 ml tubes</li>
<li>8-channel pipettor</li>
<li>filter aerosol tips</li>
<li>mol grade water</li>
<li>thermocycler</li>
</ul>
<?xml version="1.0" encoding="UTF-8"?>
<table class="table table-striped table-hover table-condensed table-responsive" style="width: auto !important; text-align: right;">
<caption>
Forward Primers
</caption>
<thead>
<tr>
<th style="text-align:left;">
Name
</th>
<th style="text-align:left;">
Sequence
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
NC1_with_Illumina_Adapter_(0N)
</td>
<td style="text-align:left;">
TCGTCGGCAGCGTCAGATGTGTATAAGAGACAGACGTCTGGTTCAGGGTTGTT
</td>
</tr>
<tr>
<td style="text-align:left;">
NC1_with_Illumina_Adapter_(1N)
</td>
<td style="text-align:left;">
TCGTCGGCAGCGTCAGATGTGTATAAGAGACAGNACGTCTGGTTCAGGGTTGTT
</td>
</tr>
<tr>
<td style="text-align:left;">
NC1_with_Illumina_Adapter_(2N)
</td>
<td style="text-align:left;">
TCGTCGGCAGCGTCAGATGTGTATAAGAGACAGNNACGTCTGGTTCAGGGTTGTT
</td>
</tr>
<tr>
<td style="text-align:left;">
NC1_with_Illumina_Adapter_(3N)
</td>
<td style="text-align:left;">
TCGTCGGCAGCGTCAGATGTGTATAAGAGACAGNNNACGTCTGGTTCAGGGTTGTT
</td>
</tr>
</tbody>
</table>
<?xml version="1.0" encoding="UTF-8"?>
<table class="table table-striped table-hover table-condensed table-responsive" style="width: auto !important; text-align: right;">
<caption>
Reverse primers
</caption>
<thead>
<tr>
<th style="text-align:left;">
Name
</th>
<th style="text-align:left;">
Sequence
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
NC2_with_Illumina_Adapter_(0N)
</td>
<td style="text-align:left;">
GTCTCGTGGGCTCGGAGATGTGTATAAGAGACAGTTAGTTTCTTTTCCTCCGCT
</td>
</tr>
<tr>
<td style="text-align:left;">
NC2_with_Illumina_Adapter_(1N)
</td>
<td style="text-align:left;">
GTCTCGTGGGCTCGGAGATGTGTATAAGAGACAGNTTAGTTTCTTTTCCTCCGCT
</td>
</tr>
<tr>
<td style="text-align:left;">
NC2_with_Illumina_Adapter_(2N)
</td>
<td style="text-align:left;">
GTCTCGTGGGCTCGGAGATGTGTATAAGAGACAGNNTTAGTTTCTTTTCCTCCGCT
</td>
</tr>
<tr>
<td style="text-align:left;">
NC2_with_Illumina_Adapter_(3N)
</td>
<td style="text-align:left;">
GTCTCGTGGGCTCGGAGATGTGTATAAGAGACAGNNNTTAGTTTCTTTTCCTCCGCT
</td>
</tr>
</tbody>
</table>
</div>
<div id="protocol-1" class="section level4">
<h4>Protocol</h4>
<ol style="list-style-type: decimal">
<li>UV treat plasticware and mol grade water for 15 min prior to PCR set-up</li>
<li>Combine 2.5 <span class="math inline">\(\mu\)</span>l of each of the forward primers at stock concentration (100 nM) to 90 <span class="math inline">\(\mu\)</span>l of mol grade water to achieve forward primer mix at 10nM working concentrationOn. Do the same for the reverse primers.</li>
<li>On ice make-up Mastermix using forward and reverse primer mixes and the recipe below.</li>
<li>Aliquot mastermix (21 <span class="math inline">\(\mu\)</span>l) to each well of 96-well plate</li>
<li>Add 4 <span class="math inline">\(\mu\)</span>l template (lysate dilution) to wells using multi-channel pipettor</li>
<li>Place microseal “A” film over completed plate and place on thermocycler (with pre-heated lid) and run the program below.</li>
</ol>
<?xml version="1.0" encoding="UTF-8"?>
<table class="table table-striped table-hover table-condensed table-responsive" style="width: auto !important; text-align: right;">
<caption>
Mastermix (25<span class="math inline">\(\mu\)</span>l)
</caption>
<thead>
<tr>
<th style="text-align:left;">
volume (<span class="math inline">\(\mu\)</span>l)
</th>
<th style="text-align:left;">
Reagent
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
5
</td>
<td style="text-align:left;">
KAPA HiFi Buffer (x5)
</td>
</tr>
<tr>
<td style="text-align:left;">
0.75
</td>
<td style="text-align:left;">
NC1 primer mix (10mM)
</td>
</tr>
<tr>
<td style="text-align:left;">
0.75
</td>
<td style="text-align:left;">
NC2 primer mix (10mM)
</td>
</tr>
<tr>
<td style="text-align:left;">
0.75
</td>
<td style="text-align:left;">
dNTPs (10mM)
</td>
</tr>
<tr>
<td style="text-align:left;">
0.5
</td>
<td style="text-align:left;">
KAPA HiFi Polymerase (0.5U)
</td>
</tr>
<tr>
<td style="text-align:left;">
13.25
</td>
<td style="text-align:left;">
mol grade water
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
1:10 dilution of lysate
</td>
</tr>
</tbody>
</table>
<p><strong>Cycling conditions</strong></p>
<ol style="list-style-type: decimal">
<li>95 for 2 min<br />
</li>
<li>98 for 20 sec<br />
</li>
<li>62 for 15 sec<br />
</li>
<li>72 for 15 sec<br />
</li>
<li>Repeat steps 2-4 25 times</li>
<li>72 for 2 min</li>
</ol>
<div class="panel panel-warning">
<div class="panel-heading">
Caution
</div>
<div class="panel-body">
<p><em>Cycles kept low to limit potential bias, but can be increased to 30-35 cycles for samples with low DNA. If amplicons are not to be cleaned immediately store at -20. It is not advisable though to delay the library preparation process as PCR products will degrade even at -20 if stored for longer than a couple of days</em></p>
</div>
</div>
</div>
</div>
<div id="magnetic-bead-purification" class="section level3">
<h3>Magnetic bead purification</h3>
<div id="reagents-required-2" class="section level4">
<h4>Reagents required</h4>
<ul>
<li>AMPure XP beads (Beckman Coulter, A63881) - Aliquot and store at <span class="math inline">\(4\,^{\circ}{\rm C}\)</span></li>
<li>96-well Magnetic-Ring stand (Applied biosystems, AM10050)</li>
<li>0.8 ml 96-well storage plates (Thermo-Scientific, AB-0859)</li>
<li>0.2 ml 96-well plate, clear, half-skirted (VWR, 83007-374)</li>
<li>8-channel pipettors</li>
<li>Reagent reservoirs</li>
<li>abosolte ethanol (mol grade)<br />
</li>
<li>mol grade water</li>
</ul>
<div class="panel panel-info">
<div class="panel-heading">
Note
</div>
<div class="panel-body">
<p><em>Filter aerosol pipette tipss and 8-channel pipettor used for majority of steps in this protocol. Also all quantities quoted are for clean-up of a full plate (96 samples)</em></p>
</div>
</div>
</div>
<div id="protocol-2" class="section level4">
<h4>Protocol</h4>
<ol style="list-style-type: decimal">
<li>Bring an aliquot of AMPure beads to room temp<br />
</li>
<li>Prepare fresh 80% ethanol from mol grade absolute ethanol (50 ml)<br />
</li>
<li>Centrifuge amplification plate to collect condensation (300 G for 2 min)<br />
</li>
<li>Remove seal from plate in bio-safety cabinet (if possible)<br />
</li>
<li>Transfer the 25 <span class="math inline">\(\mu\)</span>l reactions into 0.8 ml 96-well storage plates using multi-channel pipettor<br />
</li>
<li>Voretx beads for 30 sec to ensure even disperal before placing in reagent reservoir<br />
</li>
<li>Transfer 25 <span class="math inline">\(\mu\)</span>l of beads to each well of the storage plate and gently pipet up and down 10 times to ensure the beads are well-mixed with the PCR sample<br />
</li>
<li>Incubate at room temp for 5 min<br />
</li>
<li>Place plate on magnetic stand for 2 min (or until supernatent has cleared)</li>
<li>Once clear use multi-channel pipettor to remove and discard supernatent from the wells<br />
</li>
<li>With the plate still on the magnetic stand, wash beads with 80% ethanol by adding 200 <span class="math inline">\(\mu\)</span>l to each well (using a multi-channel pipettor and a reagent reservoir). Do not re-suspend the beads at this point<br />
</li>
<li>Incubate for 30 sec (or until clear) and then remove and discard supernatent<br />
</li>
<li>Wash again with another 200 <span class="math inline">\(\mu\)</span>l of 80% ethanol. Incubate for 30 sec and then as before remove and discard supernatant.<br />
</li>
<li>Use a small volume single-channel pipettor (P20, with fine tips) to remove excess ethanol from each well<br />
</li>
<li>Allow beads to air-dry for 15 min<br />
</li>
<li>Remove plate from magnetic stand<br />
</li>
<li>Add 40 <span class="math inline">\(\mu\)</span>l of mol grade water to each well with multi-channel pipettor and gently pipet up and down 10 times to re-suspend the beads<br />
</li>
<li>Incubate at room temp for 2 kmin<br />
</li>
<li>Place the plate back on the magnetic stand for 2 min (or until cleared)<br />
</li>
<li>Transfer approx 34 <span class="math inline">\(\mu\)</span>l of the supernatent to a new 0.2 ml 96-well plate without touching the beads<br />
</li>
<li>A microseal “A” is placed over cleaned-up samples to allow temporary storage at <span class="math inline">\(-20\,^{\circ}{\rm C}\)</span></li>
</ol>
</div>
</div>
<div id="addition-of-illumina-barcodes" class="section level3">
<h3>Addition of Illumina barcodes</h3>
<div id="reagents-required-3" class="section level4">
<h4>Reagents required:</h4>
<ul>
<li>Kapa HiFi Hotstart PCR kit with dNTPs (Kapa Biosystems, KK2502)</li>
<li>0.2 ml 96-well plates, clear, half-skirted (VWR, 83007-374)</li>
<li>Microseal “A” film (Biorad, MSA-5001)</li>
<li>1.5 ml tubes</li>
<li>8-channel pipettor</li>
<li>filter aerosol tips</li>
<li>mol grade water</li>
<li>thermocycler</li>
</ul>
<?xml version="1.0" encoding="UTF-8"?>
<table class="table table-striped table-hover table-condensed table-responsive" style="width: auto !important; text-align: right;">
<caption>
Forward primers
</caption>
<thead>
<tr>
<th style="text-align:left;">
Primer name
</th>
<th style="text-align:left;">
Sequence of primer
</th>
<th style="text-align:left;">
Row in 96-well plate
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
N501_i5
</td>
<td style="text-align:left;">
AATGATACGGCGACCACCGAGATCTACACTAGATCGCTCGTCGGCAGCGTC
</td>
<td style="text-align:left;">
A
</td>
</tr>
<tr>
<td style="text-align:left;">
N502_i5
</td>
<td style="text-align:left;">
AATGATACGGCGACCACCGAGATCTACACCTCTCTATTCGTCGGCAGCGTC
</td>
<td style="text-align:left;">
B
</td>
</tr>
<tr>
<td style="text-align:left;">
N503_i5
</td>
<td style="text-align:left;">
AATGATACGGCGACCACCGAGATCTACACTATCCTCTTCGTCGGCAGCGTC
</td>
<td style="text-align:left;">
C
</td>
</tr>
<tr>
<td style="text-align:left;">
N504_i5
</td>
<td style="text-align:left;">
AATGATACGGCGACCACCGAGATCTACACAGAGTAGATCGTCGGCAGCGTC
</td>
<td style="text-align:left;">
D
</td>
</tr>
<tr>
<td style="text-align:left;">
N505_i5
</td>
<td style="text-align:left;">
AATGATACGGCGACCACCGAGATCTACACGTAAGGAGTCGTCGGCAGCGTC
</td>
<td style="text-align:left;">
E
</td>
</tr>
<tr>
<td style="text-align:left;">
N506_i5
</td>
<td style="text-align:left;">
AATGATACGGCGACCACCGAGATCTACACACTGCATATCGTCGGCAGCGTC
</td>
<td style="text-align:left;">
F
</td>
</tr>
<tr>
<td style="text-align:left;">
N507_i5
</td>
<td style="text-align:left;">
AATGATACGGCGACCACCGAGATCTACACAAGGAGTATCGTCGGCAGCGTC
</td>
<td style="text-align:left;">
G
</td>
</tr>
<tr>
<td style="text-align:left;">
N508_i5
</td>
<td style="text-align:left;">
AATGATACGGCGACCACCGAGATCTACACCTAAGCCTTCGTCGGCAGCGTC
</td>
<td style="text-align:left;">
H
</td>
</tr>
</tbody>
</table>
<?xml version="1.0" encoding="UTF-8"?>
<table class="table table-striped table-hover table-condensed table-responsive" style="width: auto !important; text-align: right;">
<caption>
Reverse primers
</caption>
<thead>
<tr>
<th style="text-align:left;">
Primer name
</th>
<th style="text-align:left;">
Sequence of primer
</th>
<th style="text-align:left;">
Column in 96-well plate
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
N701_i7
</td>
<td style="text-align:left;">
CAAGCAGAAGACGGCATACGAGATTCGCCTTAGTCTCGTGGGCTCGG
</td>
<td style="text-align:left;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
N702_i7
</td>
<td style="text-align:left;">
CAAGCAGAAGACGGCATACGAGATCTAGTACGGTCTCGTGGGCTCGG
</td>
<td style="text-align:left;">
2
</td>
</tr>
<tr>
<td style="text-align:left;">
N703_i7
</td>
<td style="text-align:left;">
CAAGCAGAAGACGGCATACGAGATTTCTGCCTGTCTCGTGGGCTCGG
</td>
<td style="text-align:left;">
3
</td>
</tr>
<tr>
<td style="text-align:left;">
N704_i7
</td>
<td style="text-align:left;">
CAAGCAGAAGACGGCATACGAGATGCTCAGGAGTCTCGTGGGCTCGG
</td>
<td style="text-align:left;">
4
</td>
</tr>
<tr>
<td style="text-align:left;">
N705_i7
</td>
<td style="text-align:left;">
CAAGCAGAAGACGGCATACGAGATAGGAGTCCGTCTCGTGGGCTCGG
</td>
<td style="text-align:left;">
5
</td>
</tr>
<tr>
<td style="text-align:left;">
N706_i7
</td>
<td style="text-align:left;">
CAAGCAGAAGACGGCATACGAGATCATGCCTAGTCTCGTGGGCTCGG
</td>
<td style="text-align:left;">
6
</td>
</tr>
<tr>
<td style="text-align:left;">
N707_i7
</td>
<td style="text-align:left;">
CAAGCAGAAGACGGCATACGAGATGTAGAGAGGTCTCGTGGGCTCGG
</td>
<td style="text-align:left;">
7
</td>
</tr>
<tr>
<td style="text-align:left;">
N708_i7
</td>
<td style="text-align:left;">
CAAGCAGAAGACGGCATACGAGATCCTCTCTGGTCTCGTGGGCTCGG
</td>
<td style="text-align:left;">
8
</td>
</tr>
<tr>
<td style="text-align:left;">
N709_i7
</td>
<td style="text-align:left;">
CAAGCAGAAGACGGCATACGAGATAGCGTAGCGTCTCGTGGGCTCGG
</td>
<td style="text-align:left;">
9
</td>
</tr>
<tr>
<td style="text-align:left;">
N710_i7
</td>
<td style="text-align:left;">
CAAGCAGAAGACGGCATACGAGATCAGCCTCGGTCTCGTGGGCTCGG
</td>
<td style="text-align:left;">
10
</td>
</tr>
<tr>
<td style="text-align:left;">
N711_i7
</td>
<td style="text-align:left;">
CAAGCAGAAGACGGCATACGAGATTGCCTCTTGTCTCGTGGGCTCGG
</td>
<td style="text-align:left;">
11
</td>
</tr>
<tr>
<td style="text-align:left;">
N712_i7
</td>
<td style="text-align:left;">
CAAGCAGAAGACGGCATACGAGATTCCTCTACGTCTCGTGGGCTCGG
</td>
<td style="text-align:left;">
12
</td>
</tr>
</tbody>
</table>
</div>
<div id="protocol-3" class="section level4">
<h4>Protocol</h4>
<ol style="list-style-type: decimal">
<li>UV pasticware and mol grade water for 15 min prior to PCR set-up</li>
<li>On ice make-up Mastermix of water, buffer and dNTPs only (No polymerase or primers yet).</li>
<li>Aliquot mastermix (19 <span class="math inline">\(\mu\)</span>l) to each well of 96-well plate</li>
<li>Add 1.25 <span class="math inline">\(\mu\)</span>l forward primer to each well of the designated row (A-H, see table above).</li>
<li>Add 1.25 <span class="math inline">\(\mu\)</span>l reverse primer to each well of the designated column (1-12, see table above).</li>
<li>Add 0.5 <span class="math inline">\(\mu\)</span>l polyermase to each well</li>
<li>Add 3 <span class="math inline">\(\mu\)</span>l template (amplicons from the 1st round of PCR) to wells using multi-channel pipettor</li>
<li>Place microseal “A” film over completed plate and place on thermocycler (with pre-heated lid) and run the program below.</li>
<li>The resulting PCR products are cleaned-up using the magnetic bead purification protocol similiar to the 1st round of PCR (see section 3.2 above).</li>
</ol>
<?xml version="1.0" encoding="UTF-8"?>
<table class="table table-striped table-hover table-condensed table-responsive" style="width: auto !important; text-align: right;">
<caption>
Mastermix (25<span class="math inline">\(\mu\)</span>l)
</caption>
<thead>
<tr>
<th style="text-align:left;">
volume (<span class="math inline">\(\mu\)</span>l)
</th>
<th style="text-align:left;">
Reagent
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
5
</td>
<td style="text-align:left;">
KAPA HiFi Buffer (x5)
</td>
</tr>
<tr>
<td style="text-align:left;">
1.25
</td>
<td style="text-align:left;">
Forward primer, N501-508 (10mM)
</td>
</tr>
<tr>
<td style="text-align:left;">
1.25
</td>
<td style="text-align:left;">
Reverse primer, N701-712 (10mM)
</td>
</tr>
<tr>
<td style="text-align:left;">
0.75
</td>
<td style="text-align:left;">
dNTPs (10mM)
</td>
</tr>
<tr>
<td style="text-align:left;">
0.5
</td>
<td style="text-align:left;">
KAPA HiFi Polymerase (0.5U)
</td>
</tr>
<tr>
<td style="text-align:left;">
13.25
</td>
<td style="text-align:left;">
mol grade water
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
1st round PCR product
</td>
</tr>
</tbody>
</table>
<p><strong>Cycling conditions</strong></p>
<ol style="list-style-type: decimal">
<li>98 for 45 sec<br />
</li>
<li>98 for 20 sec<br />
</li>
<li>63 for 20 sec<br />
</li>
<li>72 for 2 min<br />
</li>
<li>Repeat steps 2-4 7 times</li>
</ol>
<div class="alert alert-warning" role="alert">
<p>Cycles kept low to limit potential bias</p>
</div>
</div>
<div id="individual-sample-quantification-and-pooling" class="section level4">
<h4>Individual sample quantification and pooling</h4>
<ol style="list-style-type: decimal">
<li>Nanodrop quantification of cleaned-up products resulting from second round of PCR</li>
<li>Make normalized library by combining samples into a single tube to ensure all samples are present in equal concentrations (typically combine 50 ng per sample).This should ensure uniform coverage of reads for all samples in the library.</li>
<li>Nanodrop library and dilute with mol grade water to achieve approx 5 ng/<span class="math inline">\(\mu\)</span>l</li>
</ol>
</div>
</div>
</div>
<div id="library-quantification" class="section level2">
<h2>Library Quantification</h2>
<div id="reagents-required-4" class="section level3">
<h3>Reagents required:</h3>
<ul>
<li>Illumina library Quantification Kit, Universal qPCR Mix (Kapa Biosystems, KK4824)</li>
<li>qPCR machine</li>
<li>Mol grade water</li>
<li>10 mM Tris-Hcl, pH 8 with 0.05% Tween-20</li>
<li>Multiplate PCR plate, low profile, clear, unskirted, (Biorad, MLL9601)</li>
<li>Microseal “B” seal (Biorad, MSB1001)</li>
</ul>
<div class="panel panel-info">
<div class="panel-heading">
Note
</div>
<div class="panel-body">
<p><em>When setting-up qPCR reactions always take care when pipetting small volumes and use filter aerosol pipette tips. Work on ice and use foil to prevent UV degradation of sybr green in master mix</em></p>
</div>
</div>
</div>
<div id="protocol-4" class="section level3">
<h3>Protocol</h3>
<ol style="list-style-type: decimal">
<li>Dilute library (1:1000) with 10 mM Tris-Hcl, pH 8 with 0.05% Tween-20 in triplicate</li>
<li>Using these 1:1000 dilutions set-up 1:2 serial dilutions to achieve 1:2000, 1:4000 and 1:8000 (keep in fridge until needed)</li>
<li>Before using the Illumina library Quantification kit for the first time add 1ml of primer premix (x10) to the 5ml bottle of Sybr fast qPCR Master mix (x2) and mix well</li>
<li>UV all plasticware and water for 15 min</li>
<li>Prepare qPCR plate by adding:
<ul>
<li>12 <span class="math inline">\(\mu\)</span>l Sybr fast qPCR mastermix (with primers added)<br />
</li>
<li>4 <span class="math inline">\(\mu\)</span>l mol grade water<br />
</li>
<li>and either 4 <span class="math inline">\(\mu\)</span>l diluted library, one of supplied standards (Standard 1-6) or another 4 <span class="math inline">\(\mu\)</span>l of mol grade water to act as a negative control</li>
</ul></li>
<li>Place Microseal “B” seal over reactions</li>
<li>Run on qPCR machine with the following cycling conditions: 95 for 5 min then 35 cycles of 95 for 30 sec and 60 for 45 sec</li>
<li>Look at data to confirm 90-100% reaction efficiency for samples and for standards</li>
<li>Calculate library quantification using absolute quantification against the 425 bp DNA standard</li>
</ol>
</div>
</div>
<div id="denanuration" class="section level2">
<h2>Denanuration</h2>
<div id="reagents-required-5" class="section level3">
<h3>Reagents required</h3>
<ul>
<li>Mol. grade water</li>
<li>1.5 ml eppendorf tubes</li>
<li>Ice</li>
<li>5 M NaOH</li>
<li>Vortex</li>
<li>Bench-top centrifuge</li>
<li>10 mM Tris-Cl, pH 8.5 with 0.1% Tween-20</li>
<li>Hybridization buffer (HT1, thawed and kept on ice)</li>
<li>PhiX control v3 (Illumina, FC-110-3001)</li>
</ul>
</div>
<div id="protocol-5" class="section level3">
<h3>Protocol</h3>
<ol style="list-style-type: decimal">
<li>Dilute library to 4 nM with water</li>
<li>Prepare 1ml fresh 0.2 M NaOH from 5 M NaOH stock solution</li>
<li>Denature 4 nM library:
<ol style="list-style-type: lower-roman">
<li>add 5 <span class="math inline">\(\mu\)</span>l 4 nM library with 5 <span class="math inline">\(\mu\)</span>l 0.2 M NaOH in new 1.5 ml tube</li>
<li>vortex briefly and centrifuge @ 300 g for 1 min</li>
<li>incubate at room temp. for 5 min</li>
<li>add 990 <span class="math inline">\(\mu\)</span>l hybridization buffer (chilled) to produce 20 pm library</li>
<li>place on ice</li>
</ol></li>
<li>Dilute 20 pm library to 15 pmol:
<ol style="list-style-type: lower-roman">
<li>add 900 <span class="math inline">\(\mu\)</span>l 2 0pmol library to new 1.5 ml tube</li>
<li>add 300 <span class="math inline">\(\mu\)</span>l hybridization buffer (chilled)</li>
<li>invert several times and place on ice</li>
</ol></li>
<li>Prepare 4 nM PhiX library:
<ol style="list-style-type: lower-roman">
<li>add 2 <span class="math inline">\(\mu\)</span>l 10 nM PhiX library to new 1.5 ml tube</li>
<li>add 3 <span class="math inline">\(\mu\)</span>l 10 mM Tris-Cl, pH 8.5 with 0.1% Tween-20</li>
</ol></li>
<li>Denature 4 nM PhiX library:
<ol style="list-style-type: lower-roman">
<li>add 5 <span class="math inline">\(\mu\)</span>l 4 nM PhiX library with 5 <span class="math inline">\(\mu\)</span>l 0.2 M NaOH to new 1. 5ml tube</li>
<li>vortex briefly and centrifuge @ 300 g for 1 min</li>
<li>incubate at room temp. for 5 min</li>
<li>add 990 <span class="math inline">\(\mu\)</span>l hybridization buffer (chilled) to produce 20 pM PhiX library</li>
<li>place on ice<br />
<em>This denatured 20 pM PhiX library can be stored at <span class="math inline">\(-20\,^{\circ}{\rm C}\)</span> for up to 3 weeks</em></li>
</ol></li>
<li>Dilute 20 pM PhiX library to 15 pM:
<ol style="list-style-type: lower-roman">
<li>add 225 <span class="math inline">\(\mu\)</span>l 20 pM Phix Library to new 1.5 ml tube</li>
<li>add 75 <span class="math inline">\(\mu\)</span>l hybridization buffer (chilled)</li>
<li>invert several times and place on ice</li>
</ol></li>
<li>Combine 15 pM libraries (80% sample library: 20% PhiX library):
<ol style="list-style-type: lower-roman">
<li>add 800 <span class="math inline">\(\mu\)</span>l 15 pM sample library to new 1.5 ml tube</li>
<li>add 200 <span class="math inline">\(\mu\)</span>l 15pM PhiX library</li>
<li>invert several times and place on ice until ready for loading into cartridge</li>
</ol></li>
</ol>
</div>
</div>
<div id="loading-combined-libraries" class="section level2">
<h2>Loading combined libraries</h2>
<div id="reagents-required-6" class="section level3">
<h3>Reagents required</h3>
<ul>
<li>Illumina Miseq v2 Reagent Kit, 500 cycles (MS-102-2003, contains 2 boxes)<br />
<em>Box 1, contains Reagent cartridge and hybridization buffer (HT1)- store at <span class="math inline">\(-20\,^{\circ}{\rm C}\)</span></em><br />
<em>Box 2, contains Flow cell and Incorporation buffer (RP2) - store at <span class="math inline">\(4\,^{\circ}{\rm C}\)</span></em></li>
<li>water bath</li>
<li>tweezers</li>
<li>mol grade water in sequeezy bottle</li>
<li>kim wipes</li>
<li>ethanol wipes</li>
</ul>
</div>
<div id="protocol-6" class="section level3">
<h3>Protocol</h3>
<ol style="list-style-type: decimal">
<li>Thaw cartridge in water bath at room temp. for 1 hour</li>
<li>Invert cartridge 10 times to mix reagents and bang down on bench to eliminate any visible air bubbles.</li>
<li>Wipe foil covering of position 17 before piercing with a 1 ml pipette tip.</li>
<li>Load 600 <span class="math inline">\(\mu\)</span>l of the “combined library” into the cartridge (position 17)</li>
<li>Start software and delete previous run to ensure &gt;25% memory space on machine</li>
<li>Prepare and load flow cell
<ol style="list-style-type: lower-roman">
<li>use tweezers to remove flow cell from salt solution.</li>
<li>wash-off salt with mol. grade water in a squeeze bottle, focusing on the nooks and crannies</li>
<li>dry thoroughly with kim wipe</li>
<li>use ethanol wipe to gently remove dust and grease stains from the flow cell</li>
<li>remove old flow cell from machine and place new one into position</li>
</ol></li>
<li>Load incorporation buffer and empty waste bottle into position and lower sippers</li>
<li>Load cartridge and csv sample sheet file</li>
</ol>
</div>
</div>
</div>

   </div> <!-- articleBandContent -->
</div> <!-- pageContent -->
 

<footer class="footer">
	<nav class="navbar navbar-default navbar-fixed-bottom">
	  <div class="container">
	    <div class="col-sm-12 text-center navbar-text">
	        <a rel="license" href="http://creativecommons.org/licenses/by/4.0/">
    				<img alt="Creative Commons Licence" style="border-width:0" src="https://i.creativecommons.org/l/by/4.0/88x31.png" />
    		</a>
    		This work is licensed under a <a rel="license" href="http://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution 4.0 International License</a>.
	    </div>
	  </div>
	</nav>
</footer>



</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
